cHelpers.go

```diff
@@ -1,9 +1,6 @@
 package oci8
 
-/*
-#include "oci8.go.h"
-#cgo !noPkgConfig pkg-config: oci8
-*/
+// #include "oci8.go.h"

```

Comment:
Do no remove pkg-config

Reply:
It is not removed, only the extra unneeded ones were removed. There is one left in globals.go.

Reply:
Ooops. Sorry much.

Reply:
No problem :)